libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ViewBinderCustomization.kt

```diff
@@ -181,4 +194,24 @@ class ViewBinderCustomization {
      * Use [MapViewBinder.defaultBinder] to reset to default.
      */
     var mapViewBinder: MapViewBinder? = null
+
+    /**
+     * Invoke the function to use the legacy speed limit binder.
+     * @return UIBinder of type [SpeedLimitViewBinder]
+     */
+    fun legacySpeedLimitBinder(): UIBinder? {
+        return ifNonNull(navContext) {
+            SpeedLimitViewBinder(it)
+        }
+    }
+
+    /**
+     * Invoke the function to use the default speed limit binder.
+     * @return UIBinder of type [SpeedInfoViewBinder]
```

Comment:
`SpeedInfoViewBinder` is internal, we shouldn't reference it in the docs.

Reply:
But you just removed the link. Why would we say anything here about our internal types, even if we don't reference them? Can't we just say that this UIBinder is responsible for speed info view customization or sth like that?